discord/backoff.py

```diff
@@ -22,14 +22,20 @@
 DEALINGS IN THE SOFTWARE.
 """
 
+from __future__ import annotations
+
+
 import time
 import random
+from typing import Callable, Generic, Literal, TypeVar, Type, overload, Union
+
+T = TypeVar('T', bound=Union[int, float])
```

Comment:
Shouldn't these be constrains rather than bounds?

Reply:
You're correct yes. fixed.